### PR TITLE
Adds prettier packages and updates eslint dependencies

### PR DESCRIPTION
**Pull request by madsnedergaard**
### Description

We want to improve linting and make the codebase more standardized.

This PR adds new packages for linting - we'll lint the existing codebase in a separate PR afterwards :)

Steps:
1. Add config (this PR)
2. Lint everything
3. Setup husky and lint-staged for keeping things linted going forward

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -3,7 +3,7 @@ version: 2.1
 executors:
   js-exec:
     docker:
-      - image: circleci/node:12.13
+      - image: circleci/node:12.22
```
Comment (madsnedergaard) [inline .circleci/config.yml:6]: Had to bump this for eslint to be happy

File: web/.eslintrc.js [hunk 2]
```diff
@@ -93,14 +82,10 @@ module.exports = {
     'jsx-a11y/no-static-element-interactions': 'off',
 
     // Rules we want to enable one day
-    'curly': 'off',
     'no-nested-ternary': 'off',
-    'max-len': ['off', {'code': 120}],
     'no-underscore-dangle': 'off',
-    'react/jsx-one-expression-per-line': 'off',
```
Comment (madsnedergaard) [inline web/.eslintrc.js:100]: Removes style-related rules, dunno if this conflicts with your changes @Kongkille - otherwise we can remove them here

**Comment by VIKTORVAV99** [comment]
@madsnedergaard I'd be happy to add a GitHub Action that checks prettier formatting in a separate PR after this one is merged if you want.

File: web/.eslintrc.js (hunk 2 above)
Comment (Kongkille) [reply web/.eslintrc.js:100]: I did have curly enabled on mine but we could just remove it

Comment (Kongkille) [reply web/.eslintrc.js:100]: I did add curlies to the code, did not realize it would be part of prettier :)

**Review by Kongkille — APPROVED** [review]

**Comment by madsnedergaard** [comment]
> @madsnedergaard I'd be happy to add a GitHub Action that checks prettier formatting in a separate PR after this one is merged if you want.

Thanks @VIKTORVAV99, I already have a PR in progress that adds `husky` and `lint-staged` so formatting happens before code is even committed :)

**Comment by VIKTORVAV99** [comment]
> > @madsnedergaard I'd be happy to add a GitHub Action that checks prettier formatting in a separate PR after this one is merged if you want.
> 
> Thanks @VIKTORVAV99, I already have a PR in progress that adds `husky` and `lint-staged` so formatting happens before code is even committed :)
> 
> (I'll merge your webpack5 PR first to avoid conflicts though)

Yeah I noticed but if people use github's built in editor it won't run that, probably not that big of a concern but still. The CI action would run in parallel with any existing actions so it won't really affect the current workflow. (probably should wait before the entire codebase is linted tho)

**Pull request merged**